temporian/implementation/numpy/data/display_utils.py

```diff
@@ -77,6 +77,10 @@ def display_html(evset: EventSet) -> str:
         index_data = evset.data[index_key]
         num_timestamps = len(index_data.timestamps)
         max_timestamps = config.display_max_events or num_timestamps
+        #Slices timestamps and features if there are more than 7 events
+        half_max_timestamps = (max_timestamps // 2)
+        display_timestamps = np.concatenate((index_data.timestamps[:half_max_timestamps], index_data.timestamps[-half_max_timestamps:])) if num_timestamps > max_timestamps else index_data.timestamps[:num_timestamps]
+        display_features = [np.concatenate((values[:half_max_timestamps], values[-half_max_timestamps:])) if num_timestamps > max_timestamps else values for values in index_data.features]
```

Comment:
similar thing here, if `num_timestamps > max_timestamps` doesn't hold we're reconstructing the whole list, we could just not do anything:

```python
display_features = index_data.features
if num_timestamps > max_timestamps:
    display_features = [np.concatenate((values[:half_max_timestamps], values[-half_max_timestamps:])) for values in index_data.features]

Reply:
this comment still pending - in the same way that doing `[:num_timestamps]` in the line above was unnecessary when `num_timestamps > max_timestamps` doesn't hold, reconstructing the feature list here is too. see the suggestion in comment above this one.